build.sbt

```diff
@@ -366,58 +609,63 @@ lazy val tomcatServer = libraryProject("tomcat-server")
     libraryDependencies ++= Seq(
       tomcatCatalina,
       tomcatCoyote,
-      tomcatUtilScan,
+      tomcatUtilScan
```

Comment:
Our precious trailing commas. They are good for diff reduction!

Reply:
Must be the scalafmt config? 😆

Reply:
Probably.  We've deferred running it on the build until the branches settle down, because the merges are so bad.